python/lsst/ip/isr/isr.py

```diff
@@ -345,6 +345,8 @@ def overscanCorrection(ampMaskedImage, overscanImage, fitType='MEDIAN', order=1,
         diff = numpy.abs(biasArray - medianBiasArr[:, numpy.newaxis])
         biasMaskedArr = numpy.ma.masked_where(diff > collapseRej*stdevBiasArr[:, numpy.newaxis], biasArray)
         collapsed = numpy.mean(biasMaskedArr, axis=1)
+        if collapsed.mask.sum() > 0:
+            collapsed.data[collapsed.mask] = numpy.mean(biasArray.data[collapsed.mask], axis=1)
```

Comment:
I don't know why someone would want a zero value for masked pixels, except for my particular use of this below!  I use `collapsed.data*~collapsedMask`, which provides exactly that (`collapsedMask` is `True` for bad pixels).  So the possibility is still there just in case.